share/availability/light/availability_test.go

```diff
@@ -81,38 +81,38 @@ func TestService_GetSharesByNamespace(t *testing.T) {
 		t.Run("size: "+strconv.Itoa(tt.squareSize), func(t *testing.T) {
 			getter, bServ := EmptyGetter()
 			totalShares := tt.squareSize * tt.squareSize
-			randShares := share.RandShares(t, totalShares)
+			randShares := sharetest.RandShares(t, totalShares)
 			idx1 := (totalShares - 1) / 2
 			idx2 := totalShares / 2
 			if tt.expectedShareCount > 1 {
-				// make it so that two rows have the same namespace ID
-				copy(randShares[idx2][:namespace.NamespaceSize], randShares[idx1][:namespace.NamespaceSize])
+				// make it so that two rows have the same namespace
+				copy(share.GetNamespace(randShares[idx2]), share.GetNamespace(randShares[idx1]))
 			}
 			root := availability_test.FillBS(t, bServ, randShares)
-			randNID := randShares[idx1][:namespace.NamespaceSize]
+			randNamespace := share.GetNamespace(randShares[idx1])
 
-			shares, err := getter.GetSharesByNamespace(context.Background(), root, randNID)
+			shares, err := getter.GetSharesByNamespace(context.Background(), root, randNamespace)
 			require.NoError(t, err)
-			require.NoError(t, shares.Verify(root, randNID))
+			require.NoError(t, shares.Verify(root, randNamespace))
 			flattened := shares.Flatten()
 			assert.Len(t, flattened, tt.expectedShareCount)
 			for _, value := range flattened {
-				assert.Equal(t, randNID, []byte(share.ID(value)))
+				assert.Equal(t, randNamespace, share.GetNamespace(value))
 			}
 			if tt.expectedShareCount > 1 {
 				// idx1 is always smaller than idx2
 				assert.Equal(t, randShares[idx1], flattened[0])
 				assert.Equal(t, randShares[idx2], flattened[1])
 			}
 		})
-		t.Run("last two rows of a 4x4 square that have the same namespace ID have valid NMT proofs", func(t *testing.T) {
+		t.Run("last two rows of a 4x4 square that have the same namespace have valid NMT proofs", func(t *testing.T) {
 			squareSize := 4
 			totalShares := squareSize * squareSize
 			getter, bServ := EmptyGetter()
-			randShares := share.RandShares(t, totalShares)
-			lastNID := randShares[totalShares-1][:namespace.NamespaceSize]
+			randShares := sharetest.RandShares(t, totalShares)
+			lastNID := randShares[totalShares-1][:share.NamespaceSize]
```

Comment:
Now you can use new shiny share.GetNamespace here. All other same cases could be found by looking for `:share.NamespaceSize`

Reply:
It was there before btw, but sounds good

Reply:
Done